pkg/configuration/parameters.go

```diff
@@ -101,6 +101,9 @@ type JiraAPIResourceFlags struct {
 type JiraApiResourceParametersReadIssue struct {
 }
 
+type JiraApiResourceParametersReadStatus struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type JiraApiResourceParametersReadStatus should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4860757)